02_bubble/patrickmarabeas/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Bubble implements the Bubble sort algorithm.
+func bubble(s []int) []int {
```

Comment:
nit: With the change in mutability behaviour, it would be clearer to not return anything.

Reply:
Would this result in the following signature `func bubble(s *[]int)`?

I'll add this as an additional function as the readme is expecting `func bubble(s []int) []int`.

Reply:
Added alternative implementation and tests